### ISSUE TITLE
Windows 11 Install issues.

### ISSUE THREAD

**Issue by kaboddy** [dylanaraps/neofetch#2100]
Issues with install.  I'm using OneDrive and redirecting the 'Documents' folder... Perhaps that's causing issues?  There is a local Documents folder still as well.

![2022-04-18 09_28_05-GitHub - dylanaraps_neofetch_ 🖼️ A command-line system information tool written](https://user-images.githubusercontent.com/69687422/163816430-c0b7d3bf-c033-48c3-8f5a-66b74657b1b0.png)

Path looks OK in cmd (scoop location in there).  I also restarted the Terminal just in case the path wasn't updated.

Thanks!

**Comment by Kyoto-sh** [comment]
I will also experience a similar problem after installation, I can not find the cause of the problem, I hope this will fix in the future since earlier this program worked fine

**Comment by bledavik** [comment]
Same issue here, Win11 > The system cannot find the path specified

**Comment by kaboddy** [comment]
Think I found the mistake... in the neofetch.cmd located in the 'shims' folder, there's a path to 'bash' that doesn't exist on my scoop install:
Here's the line:
`@"C:\Users\kboddy\scoop\bin\bash.exe" "C:\Users\kboddy\scoop\apps\neofetch\current\neofetch"  %*`

No bash.exe there... no bin folder either.  There is this though from the git install:
`C:\Users\kboddy\scoop\apps\git\current\bin\bash.exe`

I updated the path and it works!!!

**Comment by bledavik** [comment]
@kaboddy That fixed mine as well, good find! Thank you!

**Comment by kaboddy** [comment]
Yeah... Not sure where or why the windows installer would look for that path unless something changed with the git install.

I've got the install on another computer (Windows as well) and the files are totally different!

**Comment by Kyoto-sh** [comment]
@kaboddy Your fix helped me but after it appeared another mistake, the program simply does not work
![Screenshot_5](https://user-images.githubusercontent.com/37262044/164983265-94cb7ed9-1221-4f68-9d0f-645cd6c13a59.png)
![Screenshot_6](https://user-images.githubusercontent.com/37262044/164983266-ed0ca612-28ab-491c-afa3-24ccac538b26.png)

**Comment by kaboddy** [comment]
Not sure what to tell you.  What's your neofetch.cmd looking like?

**Comment by GrabCAD-Gary** [comment]
The problem in the cuurent version is that `neofetch.ps1` contains this:

```ps1
& $(join-path $Env:Programfiles 'Git\bin\bash.exe') $(join-path $psscriptroot 'neofetch') @args
```

which is just a rank assumption which results in `C:\Program Files\Git\bin\bash.exe`. My git is not in `C:\Program Files\`or `Git` directories. These days many people don't want apps on their smaller SSD C: drives and, whatever the cirumstances, paths shouldn't be assumed. neofetch needs to ask or detect the correct path.

The workaround is to replace `$Env:Programfiles` and `Git\` in `Git\bin\bash.exe`  with the right locations or replace the entire `$(join-path $Env:Programfiles 'Git\bin\bash.exe')` invocation with your path to bash. Something like `& 'path/to/bash.exe' $(join-path $psscriptroot 'neofetch') @args`.
